README.md

```diff
@@ -108,6 +108,17 @@ For example, the following example is invalid and will cause an error, because S
 As described above, `save.toml` is needed to configure tests. The idea is to have only one configuration file for a directory with tests (one to many relation). 
 Such directories we will call `test suites`. We decided to have only one configuration file as we have many times seen that for such tests there is a duplication of configuration in the same test suite.
 
+## Executing specific tests
+It can be useful to execute only a number of tests instead of all tests under a particular `save.toml` config.
+To do so, you want to pass a test file name:
+```bash
+$ save /path/to/tests/Test1
+```
+or a list of file names (separated with spaces)
+```bash
+$ save /path/to/tests/Test1 /path/to/tests/Test2
```

Comment:
If for example we will pass some additional arguments, but after `testFiles`
`save /path/to/tests/Test1 /path/to/tests/Test2 -parallel`

Then all of them will be interpret as test files, and there is no workaround, since we use `vararg` and can't use full name for this option?
May be we can at least notice the user, to pass arguments in this case before test files?

Reply:
Good point, added a note about order. Sadly, kotlinx.cli doesn't provide a very good help message for our case:
```bash
$ save --help
Usage: save options_list
Arguments:
    testFiles -> Paths to individual test files, can be provided to execute only them (optional) { String }
Options:
    --test-config, -c -> Path to a configuration of a test suite { String }
   ...
```
I would expect it to state something like `Usage: save options_list [files_list]`, so yes, let's at least state it in the readme